### ISSUE TITLE
Add "Related Publications" field to Create Dataset page

### ISSUE THREAD

**Issue by TaniaSchlatter** [IQSS/dataverse#3838]
We want to capture publications related to a dataset in the first step of creating a new dataset. This request is per meeting with Mercè May 9, 2017. 

See mock up attached.

![create_dataset_related_publications_field](https://cloud.githubusercontent.com/assets/15386424/26126860/340e1c6a-3a55-11e7-8593-3fa15ab0e4fe.jpg)

**Comment by landreev** [comment]
Turned the "displayoncreate" fields to TRUE, for the 4 entries in the citation block that correspond to the "related publication" field. 

To activate, the citation block needs to be re-ingested. 
The following instruction on how to do this needs to be added to the release instructions: 

We have made a minor update to the Citations metadata configuration: The "Related Publication" fields will now be included on the "Create New Dataset" page. (And not just on the "Edit Metadata" page for existing datasets. To keep your Citation metadata configuration up-to-date, re-ingest the metadata block file supplied with the Dataverse distribution, as follows: 

```
curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/citation.tsv -H "Content-type: text/tab-separated-values"
```

Also, PLEASE NOTE/REVIEW: 
The Create Dataset page will NOT match the mock up, above, 1-to-1; 
that's because  the "related publication" is not 1 field, but 3. It will be displayed the same exact way as it's been shown on the "Edit Metadata" page, for existing/already created datasets. I'm assuming this is the desired behavior.

**Comment by landreev** [comment]
pull request: https://github.com/IQSS/dataverse/pull/3841

**Comment by pdurbin** [comment]
I just reviewed pull request #3841 and it looks good, exactly the small change we talked about when estimating this issue. I'll move it to QA at https://waffle.io/IQSS/dataverse

This is out of scope, but I'd like to point out that for situations like this where we're asking users to reload a tsv file containing metadata blocks, I'd love to have a "preview" mode, which is why I opened #2551 a while back.

**Comment by djbrooke** [comment]
@TaniaSchlatter - @landreev asked if you could take a look. He'll add a screenshot here.

**Comment by landreev** [comment]
Yes, this is just to illustrate how the actual "Add new Dataset" page looks different from the mock up. The "Related Publication" is 3 fields, rather than one; and this identical to how it's rendered on the "Edit Metadata" page, for an existing dataset: 
![screen shot 2017-05-19 at 11 47 43 am](https://cloud.githubusercontent.com/assets/737558/26255960/5e1c270a-3c89-11e7-928c-26a486ecd642.png)

**Comment by dlmurphy** [comment]
I just looked into it, and this feature does no require any additions or changes to the guides.

The Related Publications feature that currently exists on the Edit Metadata page uses some really helpful tooltips, please be sure that these tooltips are also in place when the feature is added to the Add Dataset page.

**Comment by pdurbin** [comment]
@dlmurphy thanks. Also, @landreev @kcondon @mheppler @raprasad and I discussed this issue this morning and we decided to go ahead and move it to QA, which I'll do now.

**Comment by TaniaSchlatter** [comment]
Can the URL field display the width of the Citation and and notes blocks? Or, left aligned as "Vocabulary URL" above?

**Comment by pdurbin** [comment]
@pameyer pointed out that #3679 is related. Good point. Thanks.

**Comment by pdurbin** [comment]
@TaniaSchlatter it's a good idea but we estimated this issue based on the simple change made in pull request #3841 and we should be careful not to expand the scope of this (or any) issue. Can you please open a new issue about the width and alignment so it can be estimated separately? Thanks!

**Comment by TaniaSchlatter** [comment]
@pdurbin Making a UI field field consistent is not and should not be considered a "UI change." If it is, then we have and will continue to have major UI issues.

**Comment by dlmurphy** [comment]
Talked with Danny about this. The alignment fix would have been OK to work on in this issue, but since it has been closed I've made a new issue for it, #3871.
